exercises/hamming/lib/example.dart

```diff
@@ -1,7 +1,9 @@
 class Hamming {
   int distance(String a, String b) {
-    if (a.length != b.length) {
-      throw new ArgumentError("DNA strands must be of equal length.");
+    if ((a.length == 0 && b.length > 0) || (a.length > 0 && b.length == 0)) {
+      throw ArgumentError('no strand must be empty');
+    } else if (a.length != b.length) {
```

Comment:
`else if` doesn't make sense to me here. The condition `a.length != b.length` needs to be checked regardless of any prior conditions.

Suggestion: Change to independent `if` statement.
```suggestion
    } 
    
    if (a.length != b.length) {
```